README.md

```diff
@@ -266,6 +345,64 @@ const Name = observer((props: { firstName: string; lastName: string }) => {
 })
 ```
 
+## Creating MobX reactions inside hook components
+
+If needed, it is possible to create MobX based side effects in hook based components using the standard APIs. For example:
+
+```typescript
+function Counter() {
+    const store = useLocalStore(() => ({
+        count: 0,
+        inc() {
+            store.count += 1
+        }
+    }))
+
+    useEffect(
+        () =>
+            autorun(() => {
+                document.title = "Ticked: " + store.count
+            }),
+        []
+    )
+
+    return /* etc */
+}
+```
+
+Note that the disposer function of `autorun` should be returned to `useEffect` so that the effect is cleaned up properly by React.
+
+Secondly, when using MobX based side effects, you typically don't want to re-create them after each rendering, so make sure to pass in an empty array `[]` as deps to `useEffect`.
+
+This will yield the same limitation as when using `useStore`: changes to props used by the side-effect won't be picked up automatically, so don't refer to them directly. Instead, leverage `useAsObservableSource` again:
```

Comment:
```suggestion
This will yield the same limitation as when using `useLocalStore`: changes to props used by the side-effect won't be picked up automatically, so don't refer to them directly. Instead, leverage `useAsObservableSource` again:
```